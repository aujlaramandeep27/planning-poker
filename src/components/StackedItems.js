import React from 'react';

export function StackedItems({ items = [], component: Component}) {
  return (
    <div className="uk-container">
      {items.map((item, index) =>
        <div className="uk-margin" key={index}>
          <Component {...item} />
        </div>
      )}
    </div>
  )
}
