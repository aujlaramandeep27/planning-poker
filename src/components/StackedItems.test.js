import { StackedItems } from './StackedItems';
import React from 'react';

describe('Component: StackedItems', () => {
  test('should render', () => {
    expect(StackedItems({items: [{'label': 'A', 'href': 'href'}], component: props => (<div>{props.label}</div>)})).toMatchSnapshot();
  });
});
