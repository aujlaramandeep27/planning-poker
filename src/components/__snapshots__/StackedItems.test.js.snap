// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: StackedItems should render 1`] = `
<div
  className="uk-container"
>
  <div
    className="uk-margin"
  >
    <component
      href="href"
      label="A"
    />
  </div>
</div>
`;
